### PR TITLE
yaz: update to 5.30.3.

### PR DESCRIPTION
**Pull request by uw2021**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/yaz/template [hunk 1]
```diff
@@ -13,7 +13,7 @@ maintainer="Orphaned <orphan@voidlinux.org>"
 license="BSD-3-Clause"
 homepage="https://www.indexdata.com/resources/software/yaz/"
 distfiles="http://ftp.indexdata.dk/pub/yaz/yaz-${version}.tar.gz"
-checksum=311bcb386d6327dfa22e0e442aea26d1be7c633eb664b9f70d982f072a130112
+checksum=f0497fd8420574efab4e5738ea3b70787a6e8042f585156baa30bdc1911ba552
 
 case "$XBPS_TARGET_LIBC" in
 	musl) makedepends+=" libexecinfo-devel"
```
Comment (ericonr) [inline srcpkgs/yaz/template:19]: The line below should be `LDFLAGS="-lexecinfo"`

**Review by ericonr — COMMENTED** [review]
Have you managed to test with any of the dependants?

File: srcpkgs/yaz/template (hunk 1 above)
Comment (uw2021) [reply srcpkgs/yaz/template:19]: Sorry, which line are you referring to?

**Comment by uw2021** [comment]
> Have you managed to test with any of the dependants?

Yep, all working fine for me.

**Pull request closed without merging**